dvc/commands/experiments/init.py

```diff
@@ -42,6 +42,10 @@ def run(self):
         cmd = parse_cmd(self.args.command)
         if not self.args.interactive and not cmd:
             raise InvalidArgumentError("command is not specified")
+        if self.args.live and (self.args.metrics or self.args.plots):
```

Comment:
Could you clarify the reasoning for making it mutually exclusive?

Reply:
We are targeting simple use cases here, and I'm trying to mimic how we expected people to use `dvc stage add --live` before deprecation.. I think it makes sense to expect that users are either using dvclive for metrics and plots or tracking themselves. Obviously, there are use cases where people might want both, or they might be tracking images or something with dvclive that's not captured with `--live`, but I think this is better than assuming users want all of them. Otherwise, it becomes hard to do a basic `dvc exp init --live python src/train.py` scenario.

Reply:
> Otherwise, it becomes hard to do a basic dvc exp init --live python src/train.py scenario.

But if you remove it from the prompt, there is no need to make it mutually exclusive, right? 

Unless I'm missing something, if we reach this condition would be because user has explicitly passed `live` **and** one of the others